frontend/src/citizen-frontend/navigation/MobileNav.tsx

```diff
@@ -309,6 +315,7 @@ const ChildrenMenu = React.memo(function ChildrenMenu({
             data-qa={`children-menu-${child.id}`}
             to={`/children/${child.id}`}
             onClick={closeMenu}
+            alignRight
```

Comment:
This also affects everyone. But I think this is also a good change because all the other dropdowns are aligned to the right anyway.

Reply:
I think texts are already aligned right in mobile so no changes here, i hope.